dev/com.ibm.ws.jdbc_fat/publish/servers/com.ibm.ws.jdbc.fat/bootstrap.properties

```diff
@@ -12,8 +12,7 @@ com.ibm.ws.logging.trace.specification=*=info=enabled:\
 com.ibm.ws.jdbc.*=all=enabled:\
 com.ibm.ejs.j2c.*=all=enabled:\
 com.ibm.ws.rsadapter.*=all=enabled:\
-com.ibm.ws.security.jca.*=all=enabled:\
-LogService=all
+com.ibm.ws.security.jca.*=all=enabled
```

Comment:
What does this change do?

Reply:
LogService=all was logging a bunch of OSGi services that are not needed by our tace test, and are not directly related to JDBC/DB testing.  I don't think it is necessary to log, and would probably make debugging a test/product defect more difficult since we would need to weed this out anyways.

Reply:
Thanks for that answer!